--- a/cygwinccompiler.py	2021-05-03 23:11:33.000000000 +0800
+++ b/cygwinccompiler.py	2021-08-18 18:02:02.000000000 +0800
@@ -80,6 +80,8 @@
         elif msc_ver == '1600':
             # VS2010 / MSVC 10.0
             return ['msvcr100']
+        elif int(msc_ver) >= 1900 and int(msc_ver) <= 1999:
+            return ['vcruntime140']
         else:
             raise ValueError("Unknown MS Compiler version %s " % msc_ver)
 
