--- a/qt.py	2021-11-29 18:22:52.000000000 +0800
+++ b/qt.py	2021-11-29 18:23:24.000000000 +0800
@@ -609,16 +609,6 @@
 # Collect required Qt binaries, but only if all binaries in a group exist.
 def get_qt_binaries(qt_library_info):
     binaries = []
-    angle_files = ['libEGL.dll', 'libGLESv2.dll', 'd3dcompiler_??.dll']
-    binaries += _find_all_or_none(angle_files, 3, qt_library_info)
-
-    opengl_software_renderer = ['opengl32sw.dll']
-    binaries += _find_all_or_none(opengl_software_renderer, 1, qt_library_info)
-
-    # Include ICU files, if they exist.
-    # See the "Deployment approach" section in ``PyInstaller/utils/hooks/qt.py``.
-    icu_files = ['icudt??.dll', 'icuin??.dll', 'icuuc??.dll']
-    binaries += _find_all_or_none(icu_files, 3, qt_library_info)
 
     return binaries
 
